src/ImageLoaders/Xbe/XbeLoader.cs

```diff
@@ -59,7 +59,7 @@ private XbeBuildType DetectBuildType()
                 return XbeBuildType.SegaChihiro;
             }
 
-            if ((ImageHeader.EntryPoint ^ XBE_XORKEY_ENTRYPOINT_RELEASE) > ImageHeader.BaseAddress)
+            if ((ImageHeader.EntryPoint ^ XBE_XORKEY_ENTRYPOINT_RELEASE) > 0x1000000)
```

Comment:
Could you make the `0x1000000` a named constant, explaining its purpose? Remember the Reko convention is 
```C#
public const uint MagicConstant = 0x1000000
```

Reply:
Done